cmd/helpMessages.go

```diff
@@ -172,6 +172,18 @@ Log in using a VM's user-assigned identity with an Object ID of the service iden
 Log in using a VM's user-assigned identity with a Resource ID of the service identity:
 - azcopy login --identity --identity-resource-id "/subscriptions/<subscriptionId>/resourcegroups/myRG/providers/Microsoft.ManagedIdentity/userAssignedIdentities/myID"
 
+--NOTICE FOR THE FOLLOWING EXAMPLES:--
+Bear in mind that setting a environment variable from the command line will be readable in your command line history (on Windows).
+Consider clearing these entries from your history or using a small script of sorts to prompt for and set these variables.
+
+Log in as a service principal using a client secret:
+- AZCOPY_SPA_CLIENT_SECRET=secret
+- azcopy login --service-principal
+
+Log in as a service principal using a certificate & password:
+- AZCOPY_SPA_CERT_PASSWORD=password
+- azcopy login --service-principal --certificate-path /path/to/my/cert
```

Comment:
No. We cannot pull the system cert store on Windows. This has been busted since mid-2016 and has no signs of being fixed any time soon.

Reply:
Oh, what specifically is busted? (Asking because my previous project, FDT, uses the machine-level cert store all the time.  And I think I tested with the user-level one too, but only briefly, but it did work).  Is it something in the Go interface to it that's busted?

Reply:
`crypto/x509`'s SystemCertPool function is entirely broken on Windows and will not function at all.

The only PR to fix this was made mid 2016 and literally just uses powershell to pull in the cert store... This isn't preferable.

I'm strongly tempted to just go write the syscalls necessary on my own time to fix it. It's embarrassing to see incompatibilities in Go like that.

Reply:
Apologies for replying from the ol' personal account, btw. Currently working on another project and too lazy to switch.

Reply:
Ah I see.  In that case it makes sense that we do not fix or support it in AzCopy.